test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/StringSearchTestFixture.cs

```diff
@@ -21,13 +21,13 @@ public StringSearchTestFixture(DataStore dataStore, Format format, TestFhirServe
             FhirClient.DeleteAllResources(ResourceType.Patient).Wait();
 
             Patients = FhirClient.CreateResourcesAsync<Patient>(
-                p => SetPatientInfo(p, "Seattle", "Smith"),
-                p => SetPatientInfo(p, "Portland", "Williams"),
-                p => SetPatientInfo(p, "Vancouver", "Anderson"),
-                p => SetPatientInfo(p, LongString, "Murphy"))
+                    p => SetPatientInfo(p, "Seattle", "Smith", given: "Bea"),
```

Comment:
nit: Should we add an additional case where the patient either shares the family or given name? That way we can also verify if you search `name=Bea&name=Smith`, it only returns one that matches both?

Reply:
Good suggestion.